docs/source/java/vector.rst

```diff
@@ -268,6 +268,82 @@ For example, the code below shows how to build a :class:`ListVector` of int's us
      }
   }
 
+Dictionary Encoding
+===================
+
+A :class:`FieldVector` can be dictionary encoded for performance or improved memory efficiency. While this is most often done with :class:`VarCharVector`, nearly any type of vector might be encoded if there are many values, but few unique values.
```

Comment:
Maybe briefly explain dictionary encoding, or link to the glossary entry via `:term:`?

Reply:
will do

Reply:
done